public/js/reducers/chartsReducer.js

```diff
@@ -0,0 +1,97 @@
+const startVals = [{
+    data: [
+        {x: 1, y: 5},
+        {x: 2, y: 5},
+        {x: 3, y: 5},
+        {x: 4, y: 5},
+        {x: 5, y: 5},
+        {x: 6, y: 5},
+        {x: 7, y: 5}
+    ]
+}];
+
+const startAreaVals = [
+    {
+        data: [
+            {x: 1, y: 0},
+            {x: 2, y: 0},
+            {x: 3, y: 0},
+            {x: 4, y: 0},
+            {x: 5, y: 0},
+            {x: 6, y: 0},
+            {x: 7, y: 0}
+        ]
+    }, {
+        data: [
+            {x: 1, y: 100},
+            {x: 2, y: 100},
+            {x: 3, y: 100},
+            {x: 4, y: 100},
+            {x: 5, y: 100},
+            {x: 6, y: 100},
+            {x: 7, y: 100}
+        ]
+    }
+];
+
+// Creates a list of all y values from the x,y value pairs in a series' dataset
+
+const createPartialsList = (series) => {
```

Comment:
I moved these functions from the components to the reducers, which after talking to Shaun seems fine to do until the codebase grows significantly bigger. Because I'm moving the app to [jumpstate](https://github.com/jumpsuit/jumpstate), I think it makes sense to make these optimisations when that is in place